.circleci/config.yml

```diff
@@ -69,6 +69,20 @@ jobs:
       - run: sudo apt-get install -y clang-10
       - run: CC=clang-10 CXX=clang++-10 ROCKSDB_DISABLE_ALIGNED_NEW=1 USE_CLANG=1 make all -j32 # aligned new doesn't work for reason we haven't figured out
 
+  build-linux-clang10-asan:
+    machine:
+      image: ubuntu-1604:201903-01
+    resource_class: 2xlarge
+    steps:
+      - checkout # check out the code in the project directory
+      - run: pyenv global 3.5.2
+      - run: sudo sh -c 'echo "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-10 main" >> /etc/apt/sources.list'
```

Comment:
Optional: you could probably simplify to `echo "stuff" | sudo tee -a /etc/apt/sources.list`